lib/std/os/windows.zig

```diff
@@ -272,7 +272,7 @@ pub fn RtlGenRandom(output: []u8) RtlGenRandomError!void {
     const max_read_size: ULONG = maxInt(ULONG);
 
     while (total_read < output.len) {
-        const to_read: ULONG = math.min(buff.len, max_read_size);
+        const to_read = @intCast(ULONG, @min(buff.len, max_read_size));
```

Comment:
Hmm so can we do `        const to_read: ULONG = @min(buff.len, max_read_size);` now then?

Reply:
Ah yes, should work - sorry, I didn't look through your changes properly. I'm going to bed now, but I'll fix this in the morning - I'll skim through for any others when I do that, but also let me know if you spot any!

Reply:
Alright! I think this is the only one actually. I skimmed through earlier with Ctrl+F and this is all I could find.

Reply:
I took a look through - that is indeed the only one you added, but I spotted a couple more redundant `@intCast`s which I also fixed.